ee/clickhouse/queries/funnels/funnel_trends.py

```diff
@@ -145,16 +145,21 @@ def _summarize_data(self, results):
         ]
         return summary
 
-    @staticmethod
-    def _format_results(summary):
+    def _format_results(self, summary):
         count = len(summary)
         data = []
         days = []
         labels = []
 
         for row in summary:
             data.append(row["conversion_rate"])
-            days.append(row["timestamp"].strftime(HUMAN_READABLE_TIMESTAMP_FORMAT))
+            days.append(
```

Comment:
This was returning `days":["Wed. 7 Jul","Thu. 8 Jul","Fri. 9 Jul",` when it should've been like `"days":["2021-07-07","2021-07-08","2021-07-09","2021-07-10"`

Reply:
Wow, could there have been some reason or something dependent on this? Is this something that's easy to add a test for as well? Also, we should use f-strings instead of `format`.

Reply:
i don't think so, but perhaps @Twixes or @EDsCODE would know better if there was a dependent reason?

Reply:
1. Easy to add a test, though for tech debt reasons our current tests test `_exec_query` instead of `run`.
2. I think this was added by @EDsCODE when he built the MVP of funnel trends, but I don't think there was a particular reason besides preferring to format labels in the backend instead of the frontend. Definitely makes sense to return more raw data from the backend so that the frontend can do nicer things though.